source/includes/steps-create-system-dsn-linux.yaml

```diff
@@ -61,65 +63,77 @@ content: |
      * - Parameter
        - Description
 
-     * - Description
+     * - ``Description``
        - Optional. Text description of the DSN.
 
-     * - Driver
+     * - ``Driver``
        - Path to |odbc| driver file.
 
-     * - Trace
+     * - ``Trace``
        - ``On`` or ``Off``. Enable ODBC tracing.
 
-     * - Tracefile
+     * - ``Tracefile``
        - If ODBC tracing is enabled, the path to the file in which to
          store the trace output.
 
-     * - ReadOnly
+     * - ``ReadOnly``
        - ``yes`` or ``no``. Enforce a read-only ODBC session.
 
-     * - Server
+     * - ``Server``
        - Address of the server where a |bi-short|
          :binary:`~bin.mongosqld` instance is running.
 
-     * - Port
+     * - ``Port``
        - Port number of the :binary:`~bin.mongosqld` instance.
 
-     * - User
+     * - ``User``
        - Username of the MongoDB user with permission to connect to
          the :binary:`~bin.mongosqld` instance.
 
-     * - Password
+     * - ``Password``
        - Password of the MongoDB user with permission to connect to
          the :binary:`~bin.mongosqld` instance.
 
-     * - Database
+     * - ``Database``
        - Name of the MongoDB database to use.
 
-     * - SSLKEY
+     * - ``SSLKEY``
        - Full path to an SSL key file to use for establishing a secure
          connection. 
 
-     * - SSLCERT
+     * - ``SSLCERT``
        - Name of the SSL certificate file to use for establishing a
          secure connection. 
 
-     * - SSLMODE
+     * - ``SSLMODE``
        - Accepted values are:
 
-         - DISABLED
-         - PREFERRED
-         - REQUIRED
+         - ``DISABLED``
+         - ``PREFERRED``
+         - ``REQUIRED``
 
-     * - SSLCA
+     * - ``SSLCA``
        - Full path to a file with a list of trust SSL Certificate
          Authorities.
 
-     * - SSLCAPATH
+     * - ``SSLCAPATH``
        - Full path to a directory that contains trusted SSL CA
          certificates in PEM format. 
 
+     * - ``SSLCIPHER``
+       - Full path to the file containing a list of permitted
+         ciphers.
+
+     * - ``RSAKEY``
+       - Full path to the PEM file containing the RSA public key.
+
+     * - ``ENABLE_CLEARTEXT_PLUGIN``
+       - Set to ``1`` to enable cleartext authentication.
+
```

Comment:
Being that all TLS/SSL keywords are optional (depends on your config/deployment) I figured I'd add the ones we didn't have added.